.github/workflows/manual-deploy.yml

```diff
@@ -30,6 +30,8 @@ on:
         description: "Repository name. <repository>/tool:tag (Usually staphb)"
         default: "staphb"
 
+run-name: Deploy ${{ github.event.inputs.tool }} version ${{ github.event.inputs.version }}
```

Comment:
Love this addition, it makes the names of GH Action workflow much more descriptive than "Manual Deploy"

Reply:
I know! I saw it in multiqc's repo. I feel like it's going to make deploy better.